README.adoc

```diff
@@ -23,52 +23,54 @@ Refer to link:https://docs.cloudbees.com/docs/cloudbees-platform/latest/github-a
 | `name`
 | String
 | Yes
-| The name of the artifact to send to the CloudBees platform for artifact traceability purposes.
+| The name of the artifact reported to the CloudBees platform.
 
-| `version`
+| `url`
 | String
 | Yes
-| The version of the artifact to send to the CloudBees platform for artifact traceability purposes.
+| The URL where the artifact version is located.
+For example, `docker.io/myapp/myimg:1.0.0`.
 
-| `url`
+| `version`
 | String
 | Yes
-| The URL where the artifact version can be pulled for deployment.
+| The version of the artifact to send to the CloudBees platform for artifact traceability purposes.
 
 | `cloudbees-url`
 | String
 | No
 | The CloudBees platform URL. The default value is `"https://api.cloudbees.io"`.
 
-| `digest`
+| `commit`
 | String
 | No
-| The artifact digest that uniquely identifies the artifact.
+| The commit ID from the source repository.
 
-| `label`
-| String 
-| No 
-| The list of labels for the artifact.
+| `digest`
+| String
+| No, but it is recommended to help track artifact versions across repositories. 
+For more information, refer to https://docs.cloudbees.com/docs/cloudbees-platform/latest/workflows/artifacts#history[Artifact deployment and publish history].
+| The hash or checksum that uniquely identifies the artifact version.
 
-| `type`
+| `label`
 | String
 | No
-| The type of artifact, such as Docker or Maven.
+| A comma-separated list of artifact labels.
 
-| `repository-url`
+| `ref`
 | String
 | No
-| The clone URL of the repository.
+| The ref or branch of the artifact repository.
 
-| `commit`
+| `repository-url`
 | String
 | No
-| The commit ID from the source repository.
+| The artifact repository URL.
```

Comment:
`repository-url` is the clone URL of the source (code) repository so the description needs to be changed.

Reply:
will do